matlab/lib/+Ice/FileException.m

```diff
@@ -1,35 +0,0 @@
-% FileException   Summary of FileException
```

Comment:
Like in Swift, this is an unmapped exception. It's useful to test unmapped exceptions (see Ice/properties test).